src/server/services/wms/qgswmsrenderer.cpp

```diff
@@ -677,11 +677,11 @@ namespace QgsWms
 
     // configure each layer with opacity, selection filter, ...
     bool updateMapExtent = mWmsParameters.bbox().isEmpty();
-    Q_FOREACH ( QgsMapLayer *layer, layers )
+    for ( QgsMapLayer *layer : layers )
     {
       checkLayerReadPermissions( layer );
 
-      Q_FOREACH ( QgsWmsParametersLayer param, params )
+      for ( QgsWmsParametersLayer param : params )
```

Comment:
I think that `param` should be a const reference in this case.

Reply:
Can you take a look to this comment?

Reply:
It has been done, github does not squash this comment.

Reply:
I just checked on your branch and it seems that github is right. Can you retake a look? :)